analytics/pages/page_tournesol_scores.py

```diff
@@ -0,0 +1,186 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+import plotly.graph_objects as go
+import seaborn as sns
+import streamlit as st
+
+from utils import CRITERIA, CRITERI_EXT, MSG_NO_DATA, TCOLOR, api_get_tournesol_scores
+
+
+def add_expander_select_filters():
+
+    with st.expander("Select channel(s)"):
+
+        st.markdown(
+            "You can select one or several YouTube channels."
+            "If you select none, all channels will be use."
```

Comment:
We can add a space at the beginning of this string to avoid concatenating sentences.

```suggestion
            " If you select none, all channels will be use."
```